tools/make/golang.mk

```diff
@@ -41,12 +41,13 @@ go.build.multiarch: $(foreach p,$(PLATFORMS),$(addprefix go.build., $(addprefix
 
 .PHONY: go.test.unit
 go.test.unit: ## Run go unit tests
-	go test ./...
+	# The gcflags here ensure gomonkey always work, see https://github.com/agiledragon/gomonkey#notes
+	go test -gcflags=all=-l ./...
```

Comment:
im reluctant to add this,
am I'm fine with the code w/o the test (with monkey patch to mock it)

Reply:
The gomonkey can work without the no-inline flag in this case.
Could you tell me the reason why the `gcflags` is not welcome here? I always add `gcflags="all=-N -l"` with `go build` to make the debugger work usefully so I am comfortable with new gcflags...